src/commands/convertTextCommand.ts

```
@@ -0,0 +1,96 @@
+    treeLines.push('');
+    treeLines.push('*Generated by FileTree Pro Extension*');
+
+    return treeLines.join('\n');
+  }
+
+  /**
+   * Register the command with VS Code
+   * @param context - Extension context
+   * @returns Disposable
+   */
+  static register(context: vscode.ExtensionContext): vscode.Disposable {
```

ESLint

[error] 'context' is defined but never used.

(@typescript-eslint/no-unused-vars)